criu/mount.c

```diff
@@ -1109,7 +1109,7 @@ static char *get_clean_mnt(struct mount_info *mi, char *mnt_path_tmp, char *mnt_
 	}
 
 	if (mount(mi->mountpoint, mnt_path, NULL, MS_BIND, NULL)) {
-		pr_perror("Can't bind-mount %d:%s to %s",
+		pr_warn("Can't bind-mount %d:%s to %s: %m",
```

Comment:
pls replace %m on ": %s", ..., strerror(errno))". It is safer. %m will print errno of a last syscall, but pr_warn can call something else before Xprintf.

Reply:
So, what you see is v 3.

* initially, I did it by implementing `pr_pwarn`. When I decided it's not worth it for one user only, and no one else will use it.
* v2 was doing `strerror(errno)` as you are suggesting.
* v3 is what you see, I did git grep '%m[^s]' and decided it is OK to use `%m`.

Anyway, let me rollback to v2.

Reply:
Also, `\n` is missing here!

Reply:
Also, I am not fixing the error I am seeing :facepalm:

Reply:
updated